internal/suggestions/client.go

```diff
@@ -23,6 +23,19 @@ const ApiURL = "https://api.prod.speakeasyapi.dev"
 
 var baseURL = ApiURL
 
+type preprocessRequest struct {
+	Errors []string `json:"errors"`
+}
+
+type ErrorGroup struct {
```

Comment:
@chase-crumbaugh I know we're not calling this code anywhere, but what is `ErrorNumbers` arr in a single `ErrorGroup` supposed to represent?

Reply:
Oh ya a comment would have helped. It's the index of the error in the input array. The most reliable way of identifying an error (also happens to be succinct). The other option is to put the entire error string then try to match it, but after passing it through the LLM it's a bit finicky to match against